### ISSUE TITLE
Gizmo to switch to secondary weapon

### ISSUE THREAD

**Issue by sdfg2** [PeteTimesSix/SimpleSidearms#31]
Could you add a gizmo to allow all pawns within a selection to switch to secondary weapons? I usually have one lethal and one non-lethal gun depending on the raid type that's coming in, and having to go through every pawn can take up a lot of time.

**Comment by XT-0xFF** [comment]
I made an addon mod to Simple Sidearms for easier switching weapons of pawns called "Simple Sidearms - Switch Weapons", does this maybe do what you are looking for?

Link: https://steamcommunity.com/sharedfiles/filedetails/?id=2652962609

**Comment by sdfg2** [comment]
That looks like a very useful addon, but no unfortunately.  It only seems to switch from ranged to melee?  I need to switch from deadly ranged to non-lethal ranged.

**Comment by XT-0xFF** [comment]
Well, it does have an option for equipping the next sidearm, but you are a right, it does not have an explicit "lethal/non-lethal" switch. Might be an interesting idea to implement though, if there is a way to make it work and I find time to work on it.

**Comment by sdfg2** [comment]
How do you switch to the next sidearm?

**Comment by XT-0xFF** [comment]
![Screenshot 2022-04-22 200211](https://user-images.githubusercontent.com/78824040/164769710-3132cd5b-63da-494e-b39f-875af3b20ca8.png)
On the "Switch" gizmo the two white buttons on the very right allow you to move through ranged or melee weapons - depending one which is currently equipped.
To switch between ranged and melee the buttons on the left are used.

Not foolproof for what you wish, but maybe it helps.

(I wonder if there's a tag or something that I could check for if I wanted to implement a "switch to non-lethal" button. I'd imagine it to at most be added by a mod or something, so unlikely to be an option...)

**Comment by sdfg2** [comment]
Yeah, that's why I thought it would be tricky for specifically non-lethal (how do you know what is non-lethal?).  Restricting to two sidearms, one lethal, one non-lethal (using something like Compositable Loadouts), and then cycling through them should work.

The tag idea is a good one - I'm using VWE Non-Lethal (https://steamcommunity.com/sharedfiles/filedetails/?id=2454918354&searchtext=non-lethal) maybe ask to add a specific tag there that you could then use?
